lib/api-fallback.ts

```
@@ -1,13 +1,16 @@
   
   static async fetchWithFallback(
     params: URLSearchParams,
     signal?: AbortSignal
   ): Promise<Response> {
-    // Cloudflare Worker に直接接続（Vercel Function をバイパス）
-    const response = await fetch(`${this.CLOUDFLARE_ENDPOINT}?${params.toString()}`, {
+    // プレビュー環境などでは相対URLを使用
+    const isPreview = typeof window !== 'undefined' && window.location.hostname.includes('vercel.app')
+    const endpoint = isPreview ? '/api/ranking' : this.CLOUDFLARE_ENDPOINT
+    
+    const response = await fetch(`${endpoint}?${params.toString()}`, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)